docs/tutorials/control/air-quality-fleet.md

```diff
@@ -797,28 +749,247 @@ You can find all the code in the [GitHub repo for this tutorial](https://github.
+
+## Getting machines ready for third parties
+
+Continuing with our fictitious company, let's assume you want to ship air sensing machines to customers as ready-to-go as possible.
+In other words, you want to provision devices.
+
+Before an air sensing machine leaves your factory, you'd complete the following steps:
+
+1. You'd flash the single-board computer with an operating system
+2. You'd install `viam-agent`
+3. You'd provide a machine configuration template, a fragment.
+
+Once a customer receives your machine, they will:
+
+1. Plug it in and turn it on.
+2. `viam-agent` will start a WiFi network
+3. The customer uses another device to connect to the machine's WiFi network and the user gives the machine the password for their WiFi network.
+4. The machine can now connect to the internet and complete setup based on the fragment it knows about.
+
+### Create the fragment for air sensing machines
+
+In this section you will create the {{< glossary_tooltip term_id="fragment" text="fragment" >}}, that is the configuration template that all other machines will use.
+
+1. Navigate to the **FLEET** page and go to the [**FRAGMENTS** tab](https://app.viam.com/fragments).
+1. Click Create fragment.
+1. Name the fragment `air-quality-configuration`.
+1. Add the same components that you added to the development machine when you [set up one device for development](#set-up-one-device-for-development).
+
+   As a shortcut, you can use the JSON mode on the machine you already configured and copy the machine's configuration to the fragment.
+
+   {{< expand "Click here for info about the usb_interface value." >}}
+
+If you only have one USB device plugged into each of your boards, the `usb_interface` value you configured in the sensor config is likely (conveniently) the same for all of your machines.
+
+If not, you can use [fragment overwrite](/manage/fleet/reuse-configuration/#modify-fragment-settings-on-a-machine) to modify the value on any machine for which it is different.
+
+{{< /expand >}}
+
+1. Specify the version for the sds011 module.
+   At the point of writing the version is `0.2.1`.
+   Specifying a specific version or a specific minor or major version of a module will ensure that even if the module you use changes, your machines remain functional.
+   You can update your fragment at any point, any machines using it will update to use the new configuration.
+
+{{< alert title="Tip: Use the fragment on your development machine" color="tip" >}}
+To avoid differences between fragment and development machines, we recommend you remove the configured resources from the development machine and add the fragment you just created instead using the **+** button.
```

Comment:
```suggestion
To avoid differences between fragment and development machines, we recommend you remove the configured resources from the development machine, and instead use the **+** button to add the fragment you just created.
```
Slight legibility improvement. Could also try to restructure more significantly.